File: leetcode/C++/play-with-chips.cpp
class Solution {
public:
    int minCostToMoveChips(vector<int>& position) {
        int odd=0,even=0;
        for(int &n : position){
            if((n&1)==1) ++odd;
            else ++even;
        }
        return min(odd,even);
    }
};

File: leetcode/Python/play-with-chips.py
class Solution:
    def minCostToMoveChips(self, position: List[int]) -> int:
        odd, even = 0, 0
        for n in position:
            if (n&1) == 1:
                odd+=1
            else:
                even+=1
        return min(odd,even)